pombola/writeinpublic/tests.py

```diff
@@ -356,7 +511,7 @@ def test_get_form_initial(self):
     def test_get_form_kwargs_when_committee_has_multiple_emails(self):
         adapter = CommitteeAdapter()
 
-        na_committee_kind = OrganisationKind.objects.create(name='National Assembly Committees', slug='national-assembly-committees')
+        na_committee_kind = OrganisationKind.objects.create(name='National Assembly', slug='national-assembly-committees')
```

Comment:
I wish there was a better way to group organisations. that would maybe be able to deal with the fact that ad-hoc committees are also in houses.

Don't think this is the task to tackle that on though :)

Reply:
I say that because actually looking at organisation kinds, the name "... Committees" makes more sense, but I don't want to change approach at this stage.